packages/devel/libcap/package.mk

```
@@ -4,10 +4,10 @@
 # Copyright (C) 2019-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="libcap"
-PKG_VERSION="2.46"
-PKG_SHA256="4ed3d11413fa6c9667e49f819808fbb581cd8864b839f87d7c2a02c70f21d8b4"
+PKG_VERSION="2.47"
+PKG_SHA256="af165df45f9fe8b315164ec7143740947489f36ccbe6999b6cdf86e7a8dca04b"
 PKG_LICENSE="GPL"
```

```suggestion
PKG_LICENSE="GPL2"
```
`BSD 3-clause or GPL2 (you choose).`

---

Fixed.